### PR TITLE
🤖 Fix setuptools editable legacy issue

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Include SETUPTOOLS_ENABLE_FEATURES="legacy-editable" for the repo. 

### Why

There was an update, and things broke I guess. 

### How

Mostly copy/paste from duckbot. 

### Testing

I tried testing locally, but I could not re-create the issue locally for some reason. PR will do the test. 

### Additional Notes

🤞

### CODE REVIEW

File: .github/workflows/sphinx-build.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Install Dependencies
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install --editable .
+        SETUPTOOLS_ENABLE_FEATURES="legacy-editable" pip install --editable .
```
Comment (twentylemon) [inline .github/workflows/sphinx-build.yml:24]: `pip` might not be `python -m pip` -- there's no virtual environment here, so best to be specific any use `python -m pip`

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Have administrative privileges and run from cmd
 python -m venv --clear --prompt cs101 venv
 venv\Scripts\activate.bat
 pip install --upgrade pip setuptools wheel
-pip install --editable .
+SETUPTOOLS_ENABLE_FEATURES="legacy-editable" pip install --editable .
```
Comment (twentylemon) [inline README.md:28]: this usage is fine, since this is in a venv

**Review by twentylemon — APPROVED** [review]

File: .github/workflows/sphinx-build.yml (hunk 1 above)
Comment (jameshughes89) [reply .github/workflows/sphinx-build.yml]: well, it did work, but change for consistency

Comment (jameshughes89) [reply .github/workflows/sphinx-build.yml]: ok, well, I see it was reviewed already anyways

**Pull request merged**